server/fishtest/views.py

```diff
@@ -449,6 +449,8 @@ def user(request):
         request.session.flash("You cannot inspect users", "error")
         return HTTPFound(location=request.route_url("tests"))
     user_data = request.userdb.get_user(user_name)
+    if user_data is None:
+      raise exception_response(404)
```

Comment:
missing 2 spaces

Reply:
Yeah my bad, I copied the requested change. probably liked it subconsciously since 2 spaces is the correct way of indenting anyways :D